package.json

```diff
@@ -29,13 +27,13 @@
     "fastify": "^3.0.0"
   },
   "devDependencies": {
+    "@fastify/pre-commit": "2.0.2",
```

Comment:
does pre-commit also depend on pinst as husky did, or it can be dropped?

Reply:
pinst can be dropped imho

Reply:
removed pinst